plugins/woocommerce-blocks/packages/interactivity-components/dropdown/index.ts

```diff
@@ -8,15 +8,21 @@ import { getContext, store } from '@woocommerce/interactivity';
  */
 import './style.scss';
```

Comment:
the style of dropdown is broken for me. I manually deleted the build file and copy it from block folder to ensure it's not a caching issue: `trash plugins/woocommerce/assets/client/blocks/* && cp -r plugins/woocommerce-blocks/build/* plugins/woocommerce/assets/client/blocks/` 
![image](https://github.com/woocommerce/woocommerce/assets/5423135/52a086d2-3264-490e-87ec-9eecdcad6c9b)

Reply:
turned out this is a local build issue.